tasks/sep/006-1-foundation-guardian-updates/input.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
Same question as the other PR: https://github.com/ethereum-optimism/superchain-ops/pull/165/files#r1572450219

Reply:
Same response: validation of changes is sufficient to validate generation. :)